docs/source/models/concept_models/vivarium_census_synthdata/concept_model.rst

```diff
@@ -3099,22 +3099,22 @@ noise added to the data. We currently divide noise into two types:
 Noise should be added in the order below.
 
 #. "Borrowed" SSN (happens in simulation NOT noise functions)
+#. Duplications
+
+   a. Guardian-based dupilcation
```

Comment:
```suggestion
#. Guardian-based dupilcation
```
Let's not bother trying to group all duplication noise types together. If we come up with a different kind of duplication, we should consider it a separate noise type, and it may happen in a different order than implied here.